Source/Diagnostics/ReducedDiags/LoadBalanceCosts.cpp

```diff
@@ -112,9 +143,12 @@ void LoadBalanceCosts::ComputeDiags (int step)
 #else
             m_data[shift_m_data + mfi.index()*m_nDataFields + 5] = 0.;
 #endif
+            m_data[shift_m_data + mfi.index()*m_nDataFields + 6] = tbx.d_numPts(); // note: difference to volume
```

Comment:
Do we want tile box, or grown tile box? I would think the data included in ghost cells could also be relevant, but it's a diagnostic, so that's up to interpretation.

Reply:
I think for load balance, it's good to get the associated memory per box. I picked `numPts` because my understanding is that this is closer to real memory used than volume. Is that correct?

For the points in the entries above, I think using lo without the grown box is less confusing (e.g., plotting), but using real particle/cell memory size in these two new columns is what we want.

Reply:
Yeah, I would agree that you probably want to dump the tile box, not the grown box, as the description. You can grab both at the top and use as desired, as well. 🤷 

I believe num_pts is correct, yes. 

Also, if you want the raw memory of the grid points, you can look up the size of the data in the FAB at each point with something like: `const std::size_t type_size = sizeof(typename FAB::value_type);`, and then multiply. Depends on what you specifically want to output.

Reply:
Thanks for checking & confirming!

Yes, memory as well would be a good call, but I think we need to gather all multifabs that we have currently active for that (with EB, extra buffers for some algorithms, etc.), which would need a few more helpers.